xml/slert_cpuset_manipulation.xml

```diff
@@ -381,7 +381,7 @@ cset: **> [Errno 2] No such file or directory</screen>
   </sect2>
 
   <sect2 xml:id="sec-shielding-cpuset-proc-move">
-   <title>Moving Tasks with <command>proc</command></title>
+   <title>Moving tasks with <command>proc</command></title>
    <para>
     Although the ability to exec a task into a cpuset is fundamental, you
```

Comment:
OK, if `exec` is being used a lot in this doc, it poses a question. "Exec" is not a word. If it's a command, I think we should tag it with [command]exec[/command]. If they are merely using it as shorthand for "execute" then we should either use the word in full, or a shorter one, for example "run".

Thoughts?

Reply:
I share those doubts and think it's a worthwhile future topic. I'd also avoid doing that in this PR, if we can. Otherwise, we'll end up fixing more than I am ready for.